PrebidMobileTests/RenderingTests/Tests/Prebid/PBMORTBTest.swift

```diff
@@ -23,11 +23,10 @@ class PBMORTBTest: XCTestCase {
     // MARK: - ORTB Response
     
     func testBidToJsonString() {
-        let bid = PBMORTBBid<NSDictionary>()
```

Comment:
Unfortunately, we can't modify the support of OpenRTB protocol in this way. Because it is just a protocol and there actually no guarantees that some required field will be missed  in the real responses. 

Yes the probability is very low, but it still exists. Please make all properties of OpenRTB optional. And resubmit the PR. 

You can also remove PBM prefix for all new Swift classes, if it not break another PR related to parsing.